core/services/privacy_policy/PrivacyPolicyInterface.php

```diff
@@ -7,18 +7,19 @@
 
 
 /**
- * Class PolicyGeneratorInterface
+ * Class PrivacyPolicyInterface
  * Description
```

Comment:
just remove this if you are not going to write something for a description

Reply:
oups! I'll descrive it